spec/models/organization_spec.rb

```diff
@@ -132,16 +133,19 @@
         end
 
         it "retrieves the distributions scheduled for this week that have not yet happened" do
-          wednesday_distribution_scheduled = create(:distribution, organization: @organization, state: :scheduled, issued_at: Time.zone.local(2019, 7, 3))
-          create(:distribution, organization: @organization, state: :complete, issued_at: Time.zone.local(2019, 7, 3))
-          sunday_distribution = create(:distribution, organization: @organization, state: :scheduled, issued_at: Time.zone.local(2019, 7, 7))
-          upcoming_distributions = @organization.distributions.upcoming
+          wednesday_distribution_scheduled = create(:distribution, organization: organization, state: :scheduled, issued_at: Time.zone.local(2019, 7, 3))
+          create(:distribution, organization: organization, state: :complete, issued_at: Time.zone.local(2019, 7, 3))
+          sunday_distribution = create(:distribution, organization: organization, state: :scheduled, issued_at: Time.zone.local(2019, 7, 7))
+          upcoming_distributions = organization.distributions.upcoming
           expect(upcoming_distributions).to match_array([wednesday_distribution_scheduled, sunday_distribution])
         end
       end
     end
 
     describe "items" do
```

Comment:
These tests take ages. I wonder if anything can be done to speed them up?

Reply:
Do you know what it's spending the time on? Often I find there's some heavy setup, which can be helped just by combining a bunch of assertions into one test.

Reply:
It seem like the setup step takes a while. Putting the setup in a `before(:all)` cut down the time by 60% from 8s to about 3. 

Good enough for now imo but maybe it can be changed to not seed all the items and stick to idk 10?